src/app/pages/send/choose-crypto-asset/choose-crypto-asset.tsx

```diff
@@ -34,9 +41,33 @@ export function ChooseCryptoAsset() {
 
   useRouteHeader(<ModalHeader hideActions defaultGoBack title=" " />);
 
+  function navigateToSendForm({
+    cryptoAssetBalance,
+  }: {
+    cryptoAssetBalance: AllTransferableCryptoAssetBalances;
+  }) {
```

Comment:
Any reason for this to be a obj rather than single prop?

Reply:
imho using objects for arguments give function calls more readability, but i can change it to a single prop